src/hex_boosting.proto

```diff
@@ -31,3 +33,11 @@ message boosted_hex_update_v1 {
   // Details of the updated hex
   boosted_hex_info_v1 update = 2;
 }
+
+enum boosted_hex_device_type_v1 {
```

Comment:
nit: can this enum be moved to the top of the file.  Things read better when sub types are defined prior to their use